packages/frontend/src/components/UserSettings/AppearanceSettingsPanel/index.tsx

```diff
@@ -0,0 +1,503 @@
+import { subject } from '@casl/ability';
+import {
+    ActionIcon,
+    Button,
+    Card,
+    ColorInput,
+    ColorSwatch,
+    Flex,
+    Group,
+    Loader,
+    Popover,
+    SimpleGrid,
+    Stack,
+    Text,
+    Title,
+    Tooltip,
+} from '@mantine/core';
+import { useForm } from '@mantine/form';
+import {
+    IconBolt,
+    IconChevronDown,
+    IconColorFilter,
+    IconCubeUnfolded,
+    IconInfoCircle,
+    IconLeaf,
+    IconRestore,
+    IconRipple,
+    IconSunglasses,
+} from '@tabler/icons-react';
+import { isEqual } from 'lodash';
+import { FC, useCallback, useEffect, useState } from 'react';
+import { useOrganization } from '../../../hooks/organization/useOrganization';
+import { useOrganizationUpdateMutation } from '../../../hooks/organization/useOrganizationUpdateMutation';
+import { isHexCodeColor } from '../../../utils/colorUtils';
+import { Can, useAbilityContext } from '../../common/Authorization';
+import MantineIcon from '../../common/MantineIcon';
+import { SettingsCard } from '../../common/Settings/SettingsCard';
+
+interface ColorPalette {
+    name: string;
+    colors: string[];
+    icon: typeof IconBolt;
+}
+
+const defaultColorPalettes: ColorPalette[] = [
```

Comment:
we could move this to a separate file, even json. But we can keep it here for now